pkg/network/node/pod.go

```diff
@@ -160,13 +161,39 @@ func (m *podManager) Start(rundir string, localSubnetCIDR string, clusterNetwork
 	return m.cniServer.Start(m.handleCNIRequest)
 }
 
+func (m *podManager) InitRunningPods(existingPodSandboxes map[string]*kruntimeapi.PodSandbox, existingOFPodNetworks map[string]podNetworkInfo) error {
+	m.runningPodsLock.Lock()
+	defer m.runningPodsLock.Unlock()
+	defer glog.V(5).Infof("Finished initializing podManager with running pods at start-up")
```

Comment:
Don't use `defer`. Just log it at the end.

Reply:
Any reason in particular for this? Performance? I see we are using defer when unlocking the runningPodsLock at every invocation elsewhere in the code. Should I modify those too in that case?